src/lib/ndpi_content_match.c.inc

```diff
@@ -1630,6 +1633,7 @@ static ndpi_tls_cert_name_match tls_certificate_match [] = {
   { "O=Riot Games Inc",                NDPI_PROTOCOL_RIOTGAMES   },
   { "O=NoMachine",                     NDPI_PROTOCOL_NOMACHINE },
   { "O=Private Internet Access",       NDPI_PROTOCOL_PIA },
+  { "CN=*.adobeconnect.com",           NDPI_PROTOCOL_ADOBE_CONNECT },
```

Comment:
There are no strict rules, but I tend to prefer to avoid adding entries in this list if not really necessary; I do think that all the Adobe TLS traffic will have a SNI for domain matching